config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
**IF** it is a problem: `'debug' => strtolower(env('DEBUG')) === 'true' ? true : false,`